scene/resources/packed_scene.h

```diff
@@ -143,6 +143,7 @@ class SceneState : public RefCounted {
 	String get_path() const;
 
 	void clear();
+	virtual Error copy_from(const Ref<SceneState> &p_scene_state);
```

Comment:
Why is this virtual? Sounds like it was copied from `Resource`'s `virtual` one but `SceneState` doesn't inherit `Resource`.